types/rebass__forms/index.d.ts

```diff
@@ -0,0 +1,54 @@
+// Type definitions for @rebass/forms 4.0
+// Project: https://github.com/rebassjs/rebass#readme
+// Definitions by: zinozzino <https://github.com/zinozzino>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.1
+
+import { ResponsiveStyleValue, SystemStyleObject } from '@styled-system/css';
```

Comment:
This one is not used, can be removed I think

Reply:
Thank you! I just remove those imports.